docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters/create-single-region.md

```diff
@@ -65,31 +65,33 @@ Select **Single-Region Deployment** and set the following options.
 
 Fault tolerance determines how resilient the cluster is to node and cloud zone failures. Choose one of the following:
 
+- **Availability Zone Level** - Minimum of 3 nodes spread across multiple availability zones with a [replication factor](../../../../architecture/docdb-replication/replication/) (RF) of 3. YugabyteDB can continue to do reads and writes even in case of a cloud availability zone failure. This configuration provides the maximum protection for a data center failure. Recommended for production deployments. For horizontal scaling, nodes are scaled in increments of 3.
```

Comment:
The list in the previous page has been changed from 

- **UI Element** - Description

to

- **UI Element**: Description

format